schema/vrs/def/SequenceReference.rst

```diff
@@ -82,6 +82,16 @@ Some SequenceReference attributes are inherited from :ref:`gks-core:Entity`.
       - string
       - 0..1
       - The interpretation of the character codes referred to by the refget accession, where "aa" specifies an amino acid character set, and "na" specifies a nucleic acid character set.
+   *  - sequence
+      - 
+      - {'$ref': '#/$defs/sequenceString'}
+      - 0..1
+      - A :ref:`SequenceString` that is a literal representation of the referenced sequence.
```

Comment:
This should be lowercase `sequenceString`...

Reply:
good catch!